src/config/tajikistan/layers.json

```diff
@@ -319,6 +319,11 @@
       "id": "wp_pop_cicunadj",
       "key": "label"
     },
+    "zonal": {
+      "class_properties": [
+        "label"
```

Comment:
If this is the default for all, should we just add it as the default and not need it in the config?

Reply:
good idea. done with https://github.com/WFP-VAM/prism-frontend/pull/431/commits/750abe11ebe142022333b40dae39f891bef602e5